sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -0,0 +1,427 @@
+   "source": [
+    "customers_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=True\n",
+    ")\n",
+    "\n",
+    "orders_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To confirm that your FeatureGroup has been created we use `DescribeFeatureGroup` and `ListFeatureGroups` APIs to display the created FeatureGroup."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders_feature_group.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sagemaker_session.boto_session.client('sagemaker', region_name=region).list_feature_groups() # We use the boto client to list FeatureGroups"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Ingest data into a feature group\n",
+    "\n",
+    "After the FeatureGroups have been created, we can put data into the FeatureGroups by using the `PutRecord` API. It will take < 1min to ingest data both of these FeatureGroups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def check_feature_group_status(feature_group):\n",
+    "    status = feature_group.describe().get(\"FeatureGroupStatus\")\n",
+    "    while status == \"Creating\":\n",
+    "        print(\"Waiting for Feature Group to be Created\")\n",
+    "        time.sleep(5)\n",
+    "        status = feature_group.describe().get(\"FeatureGroupStatus\")\n",
+    "    print(f\"FeatureGroup {feature_group.name} successfully created.\")\n",
+    "    \n",
+    "check_feature_group_status(customers_feature_group)\n",
+    "check_feature_group_status(orders_feature_group)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.ingest(\n",
+    "    data_frame=customer_data, max_workers=3, wait=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders_feature_group.ingest(\n",
+    "    data_frame=orders_data, max_workers=3, wait=True\n",
+    ")"
+   ]
+  },
+  {
```

Comment:
Before cleaning up, shall we call GetRecord once to show that data has been ingested into feature group?